packages/server/src/bandit/banditSelection.ts

```diff
@@ -11,7 +11,7 @@ import { logError } from '../utils/logging';
 //const EPSILON = 0.1;
 
 // NULL HYPOTHESIS - always pick at random
-const EPSILON = 1;
+// const EPSILON = 1;
```

Comment:
maybe let's remove the commented out stuff, since we've completed the null hypothesis already